docs/cni/kubernetes/manifests/calico-configmap.yaml

```diff
@@ -0,0 +1,33 @@
+# This ConfigMap can be used to configure a self-hosted Calico installation.
+# See `calico-hosted.yaml` for an example of a Calico deployment which uses
+# the config in this ConfigMap.
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: calico-config 
+  namespace: kube-system
+data:
+  # Configure this with the location of your etcd cluster.
```

Comment:
Hoping that we can eventually reference from here, but can't at the moment.

I'll update the comment.